src/main/java/org/rumbledb/compiler/TranslationVisitor.java

```diff
@@ -105,11 +105,8 @@
 public class TranslationVisitor extends org.rumbledb.parser.JsoniqBaseVisitor<Void> {
```

Comment:
This comment is orthogonal to the content of this PR, but I thought it could be addressed here.

Since we have seen quite a bit of refactoring in this area, I was wondering if we should add a few lines of comments here explaining what "Translation" in this context is.

I have seen the comment in the line above "// used to build AST, will override methods" but did not find it very useful and even potentially incorrect. If I am not mistaken, isn't this visitor used to build the Node(previously ExpressionOrClause) tree. AST should already be generated by the ANTLR parser. What do you think?

Reply:
Great idea. DONE.